bfh/transformations.py

```diff
@@ -367,7 +367,9 @@ class DateToIsoString(Transformation):
     """
     def function(self, source, *call_args):  # source ignored
         try:
-            return call_args[0].isoformat()
+            argument = call_args[0]
```

Comment:
There's already such a mad jumble between the init args to this descriptor and the runtime 'source' arg (an arrangement that is totally my fault, sorry!) that it would be nice to not call this variable 'argument'... could you please follow the pattern seen elsewhere in this file and call that variable 'value'?